### ISSUE TITLE
Is it possible to turn off WM effects when in Energy Saver Mode

### ISSUE THREAD

**Issue by rahmanshaber** [martin31821/cpupower#122]
I think it will be better to turn off the fancy WM or other effects when Energy Saver mode is turned on.
That will save little bit more power.

**Comment by fin-ger** [comment]
Currently this is not planned, however it seems like a good addition to #14

**Comment by rahmanshaber** [comment]
yes, that reducing display brightness is also a very good option.

If don't mind asking, are doing this things using script or lib/functions from the Gnome it self.
